recipes/abawaca/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set name = "abawaca" %}
+{% set version = "1.00" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/CK7/abawaca/archive/v{{ version }}.tar.gz
+  sha256: 57465bb291c3a9af93605ffb11d704324079036205e5ac279601c9e98c467529
+
+build:
+  number: 0
+
+requirements:
+  build:
+
```

Comment:
You are compiling from source here, correct? Please include gcc, llvm as stated in our Guidelines.